conans/util/files.py

```diff
@@ -55,6 +57,17 @@ def is_dirty(folder):
     return os.path.exists(dirty_file)
 
 
+@contextmanager
+def set_dirty_context_manager(folder, force_clean=False):
```

Comment:
force_clean is never set to True, so it can be removed.

Reply:
yes, you should be. Cleaning something broken is a prone error

Reply:
done